base/algorithms.jl

```diff
@@ -0,0 +1,119 @@
+module Algorithms
```

Comment:
I agree "algorithm" is a bit vague (I "algorihtms" like in the c++ STL...), but no problem I will update and follow existing practices here.